[IMP] Payroll: Saudi Arabia Localization 19.3 (#18464)
odoo/documentation: content/applications/hr/payroll/payroll_localizations/saudi_arabia.rst

.. image:: saudi_arabia/check-payslips-sa.png
   :alt: The worked days tab of a payslip.

> Annual leave Provision calculation

Reviewer: ```suggestion
Annual leave provision calculation
```

one third of the |EOS| amount that would normally be granted in the case of an *End of Contract*.

.. example::

>    If an employee worked for thee years and resigned, the |EOS| is first computed as if the

Reviewer: ```suggestion
   If an employee worked for three years and resigned, the |EOS| is first computed as if the
```

> automatically added on the paysliplines to account for the difference between the provisioned amount

Reviewer: ```suggestion
automatically added on the payslip lines to account for the difference between the provisioned amount
```

and the actual amount due.

> Leaves advance pay

Reviewer: ```suggestion
Leave advance pay
```
i believe im fixing a grammatical error but im not sure if the term is incorrect now

Author: They use the term "Leaves" instead of "vacation" or "time off", so since that's the vernacular they use, I copied it - plus it is how they had written it as well. I wil change this to "Advanced leave Pay" then it's gramatically correct AND it makes more sense.
